rmgpy/rmg/reactTest.py

```diff
@@ -100,7 +101,7 @@ def testReactAll(self):
                 ]
 
         N = len(spcs)
-        rxns = react_all(spcs, N, np.ones(N), np.ones([N, N]), np.ones([N, N, N]))
+        rxns = react_all(spcs, N, np.ones(N), np.ones([N, N]), 2, np.ones([N, N, N]))
```

Comment:
This also needs to be updated following the change in argument position.

Reply:
Done, but still haven't found a solution for the error in liquidTest.py due to "fixing" the thermo labeling.